src/nikhilMJun20/CharClassify.java

```diff
@@ -0,0 +1,39 @@
+/*Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2
+*/
+
+package nikhilMJun20;
+
+public class CharClassify {
+	void findCharType(String str) {
+		int digitCnt=0, upperCnt=0, lowerCnt=0, specialCnt=0;
+		
+		for(int index=0; index<str.length(); index++) {
+			if(Character.isDigit(str.charAt(index)))
+				digitCnt++;
+			else if(Character.isLetter(str.charAt(index))) {
+				if(Character.isLowerCase(str.charAt(index)))
+					lowerCnt++;
```

Comment:
But here as I said you don't need to right else if and if and else block. 
if(Character.isDigit(ch))
				dCount++;
			else if(Character.isUpperCase(ch))
				uCount++;
			else if(Character.isLowerCase(ch))
				lcCount++;
			else
				sCount++;
No need to check for letters, because it will aready going to cover under uppercase and lowercase.

Reply:
Hi Krishna,
This is the most elegant solution.
Thank you for a very thorough review and comments for improvement.
I have made the changes. Please review and merge. Thanks!